andela/wg-apollo
# #152919354 Fitbit Support

#### What does this PR do

Adds Fitbit integration support.

#### Description of the task to be completed

Being able to pull in exercise and weight information from fitbit would be fantastic. The upside of intergrating with Fitbit is that they already allow you to import data from places like calorie counter.

https://dev.fitbit.com/docs/

Fitbit provides an open API, not like myfitnesspal that has a closed API. In the beginning we should start with something easy like the body weight and look at food logging once our own is ready

#### How should this manually be tested?

* Log in to your wger account
* Click on the `Weight` dropdown link and click on the `Weight Overview` link
* Click on the `Options` button link and choose `Import from Fitbit`
* Click on the `Grant Wger Access To Your Fitbit Data` button
* Log in to your Fitbit account
* Click on `Allow` to grant access to Fitbit
* Navigate back to `Weight Overview` to see the synced data

#### Any background context you want to provide?

#### Pivotal Tracker story

See [Feature #152919354](https://www.pivotaltracker.com/story/show/152919354)

#### Screenshots

<img width="1358" alt="screen shot 2017-12-08 at 12 46 25" src="https://user-images.githubusercontent.com/31339738/33760886-60239e50-dc18-11e7-8f24-d87b828563dd.png">
<img width="1358" alt="screen shot 2017-12-08 at 12 46 53" src="https://user-images.githubusercontent.com/31339738/33760905-6d53e42c-dc18-11e7-9838-1ecc738d62a2.png">
<img width="1357" alt="screen shot 2017-12-08 at 12 47 58" src="https://user-images.githubusercontent.com/31339738/33760918-76717ab0-dc18-11e7-8b4f-486faeae8b65.png">
<img width="496" alt="screen shot 2017-12-08 at 12 48 17" src="https://user-images.githubusercontent.com/31339738/33760934-7e7e2d66-dc18-11e7-8b4c-02cccbae1d2d.png">
<img width="1357" alt="screen shot 2017-12-08 at 12 48 42" src="https://user-images.githubusercontent.com/31339738/33760939-84f89f32-dc18-11e7-9fbf-e1e7fd70d873.png">

#### Questions
